app/src/main/java/com/swent/mapin/ui/map/BottomSheetContent.kt

```
@@ -137,13 +139,19 @@ fun BottomSheetContent(
     onMemorySave: (MemoryFormData) -> Unit = {},
     onMemoryCancel: () -> Unit = {},
     onTabChange: (MapScreenViewModel.BottomSheetTab) -> Unit = {},
-    onTabEventClick: (Event) -> Unit = {},
-    onProfileClick: () -> Unit = {}
+    onJoinedEventClick: (Event) -> Unit = {},
+    onProfileClick: () -> Unit = {},
+    filterViewModel: FiltersSectionViewModel = viewModel(),
+    locationViewModel: LocationViewModel = viewModel(),
+    profileViewModel: ProfileViewModel = viewModel()
+    onTabEventClick: (Event) -> Unit = {}
```

detekt (1.23.8)

[warning] This empty block of code can be removed.

(detekt.empty-blocks.EmptyFunctionBlock)